web/src/main/java/org/mskcc/cbio/portal/web/ProxyController.java

```diff
@@ -86,6 +86,25 @@ public void setEnableOncokb(Boolean property) {
         }
         this.enableOncokb = property;
     }
+
+  private String civicURL;
+  @Value("${civic.url:https://civic.genome.wustl.edu/api/}")
+  public void setCivicURL(String property) {
+      if (property.isEmpty()) {
```

Comment:
I believe the value annotation `@Value("${civic.url:https://civic.genome.wustl.edu/api/}")` gives `property` a default value of `https://civic.genome.wustl.edu/api/` in this case, so property will never be empty and you can simplify this code.